repos/system_upgrade/el7toel8/actors/peseventsscanner/libraries/library.py

```diff
@@ -139,9 +143,12 @@ def scan_events(path):
     """ Scan JSON file containing PES events """
     try:
         events = parse_file(path)
-    except (ValueError, KeyError) as error:
-        raise StopActorExecutionError(
-            message='Found invalid or outdated PES data.',
-            details={'details': error, 'hint': 'Consult docs to retrieve updated data.'})
+    except (ValueError, KeyError) as value_error:
+        title = 'Missing PES data ({})'.format(path)
```

Comment:
I think "Missing" is not the right word here. You can reach this with an invalid data as much as for missing. We will explain better for the user what the error is in details. With that I would go with something like "Missing/Invalid PES data file", or anything else you think it is better.